vendor/experimental-render-mode-rehydrate.js

```diff
@@ -0,0 +1,25 @@
+(function() {
+  if (typeof FastBoot === 'undefined') {
+    var current = document.getElementById('fastboot-body-start');
+
+    if (current && Ember.ViewUtils.isSerializationFirstNode(current.nextSibling)) {
+      Ember.ApplicationInstance.reopen({
```

Comment:
Don't we need to turn off this: https://github.com/ember-fastboot/ember-cli-fastboot/blob/master/app/instance-initializers/clear-double-boot.js ?

Reply:
The clear-double-boot checks to see if `fastboot-body-start` is present.  If we detect true with `Ember.ViewUtils.isSerializationFirstNode` we eventually delete `fastboot-body-start` which prevents the clear-double-render.